clearlinux/Dockerfile

```diff
@@ -1,50 +0,0 @@
-#
```

Comment:
Can we please all the existing files and move them to `2.1/run/<distro>` instead ?
I know they're broken, but if we remove them we will have a hard time fixing them...